pocs/version.py

```diff
@@ -1,6 +1,6 @@
 # this file was automatically generated
```

Comment:
Is this claim true? If so, how is it generated?

Reply:
Having now read [Creating a New Release](https://github.com/panoptes/POCS/wiki/Creating-a-New-Release), I can see that this file manually updated, so the comment should be removed.

Reply:
Amazing how I can miss that in a file that has 5 lines total. Thanks.